packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeRate.ts

```diff
@@ -114,6 +128,52 @@ export const getTradeRate = async (
     return { networkFeeCryptoBaseUnit: fast.txFee }
   }
 
+  const buyAssetAddress =
+    buyAsset.assetId === solAssetId
+      ? fromAssetId(wrappedSolAssetId).assetReference
+      : fromAssetId(buyAsset.assetId).assetReference
+
+  const sellAssetAddress =
+    sellAsset.assetId === solAssetId
+      ? fromAssetId(wrappedSolAssetId).assetReference
+      : fromAssetId(sellAsset.assetId).assetReference
+
+  const [buyAssetAccount] = await PublicKey.findProgramAddressSync(
+    [
+      Buffer.from('referral_ata'),
+      new PublicKey(SHAPESHIFT_JUPITER_REFERRAL_KEY).toBuffer(),
+      new PublicKey(buyAssetAddress).toBuffer(),
+    ],
+    new PublicKey(JUPITER_AFFILIATE_CONTRACT_ADDRESS),
+  )
+
+  const [sellAssetAccount] = await PublicKey.findProgramAddressSync(
```

Comment:
And same here

Reply:
Changed in [6840b5c](https://github.com/shapeshift/web/pull/8267/commits/6840b5c3ff9eefd79239117bbd0c17cdb1c4bebf)